stixcore/idb/manager.py

```diff
@@ -237,27 +237,37 @@ def convert_mib_2_sqlite(*, in_folder, out_file, version_label):
                         nextID = nr
                 nextID += 1
 
-                # NIX00405
-                nix = 'NIX00405'
-                count,  = cur.execute("select count(*) from PCF where PCF_NAME = ? " +
-                                      "AND PCF_CURTX not NULL", (nix,))\
-                             .fetchone()
-                if count == 0:
-                    pname, nextID = IDB.generate_calibration_name("CIX", nextID)
-                    cur.execute('''update PCF set
-                                    PCF_CURTX = ?,
-                                    PCF_CATEG = 'N',
-                                    PCF_UNIT = 's'
-                                   where PCF_NAME = ?''', (pname, nix))
-
-                    cur.execute('''insert into MCF (MCF_IDENT, MCF_DESCR, MCF_POL1,
-                                    MCF_POL2, MCF_POL3, MCF_POL4, MCF_POL5, SDB_IMPORTED)
-                                   values
-                                    (?,?,?,?,?,?,?, 0)''', (pname, "Duration", 0, 0.1, 0, 0, 0))
-
-                else:
-                    logger.info(f"Skip calib injection for {nix}: allready present")
-                print(nix)
+                # inject polynomial calibrations
+
+                duration = ("duration", 0, 0.1, 0, 0, 0)
+                duration_p1 = ("duration + 0.1", 0.1, 0.1, 0, 0, 0)
+                binary_seconds = ("binary seconds", 0, 1.0 / 65536, 0, 0, 0)
+
+                for nix, config, unit in [('NIX00269', duration, "s"),
+                                          ('NIX00441', duration, "s"),
+                                          ('NIX00122', duration, "s"),
+                                          ('NIX00124', duration, "s"),
+                                          ('NIX00404', duration_p1, "s"),
+                                          ('NIX00405', duration_p1, "s"),
+                                          ('NIX00123', binary_seconds, "s")]:
```

Comment:
So it was my understanding that NIX00404 the conversation is (NIX00404)*0.1 and for NIX00405 it is (NIX00405+1)*0.1  As NIX00404 could be 0 while a value of 0 NIX00405 doesn't really make sense but I'm not sure.

Reply:
as discussed with Laszlo we will go with the 'NIX00405' as duration and add the 0.1 only in the QL data as specified in the IDB document.
Might change later